rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -76,6 +76,53 @@ def test_exec(config, ssh_con):
     else:
         all_users_info = s3lib.create_users(config.user_count)
 
+    if config.multisite_sync_policy:
+        ceph_version_id, _ = utils.get_ceph_version()
+        ceph_version_id = ceph_version_id.split("-")
+        ceph_version_id = ceph_version_id[0].split(".")
+        if float(ceph_version_id[0]) >= 16:
+            if utils.is_cluster_multisite():
+                cmd = (
+                    "radosgw-admin sync group create --group-id=group1 --status=enabled"
+                )
+                out = utils.exec_shell_cmd(cmd)
+                cmd = "radosgw-admin period get"
+                period_list = utils.exec_shell_cmd(cmd)
+                period_list = json.loads(period_list)
+                zone_names = ""
+                for i in range(
+                    len(period_list.get("period_map")["zonegroups"][0]["zones"])
+                ):
+                    zone_name = period_list.get("period_map")["zonegroups"][0]["zones"][
+                        i
+                    ]["name"]
+                    zone_names = zone_names + zone_name
+                    if (
+                        i
+                        != len(period_list.get("period_map")["zonegroups"][0]["zones"])
+                        - 1
+                    ):
+                        zone_names = zone_names + ","
+                cmd = f"radosgw-admin sync group flow create --group-id=group1 --flow-id=group1-flow --flow-type=symmetrical --zones={zone_names}"
+                out = utils.exec_shell_cmd(cmd)
+                cmd = "radosgw-admin sync group pipe create --group-id=group1 --pipe-id=group1-pipe --source-zones='*' --source-bucket='*' --dest-zones='*' --dest-bucket='*'"
+                out = utils.exec_shell_cmd(cmd)
+                realm_name = period_list.get("realm_name")
+                cmd_realm = (
+                    f"radosgw-admin period update --rgw-realm={realm_name} --commit"
+                )
+                out = utils.exec_shell_cmd(cmd_realm)
+                cmd = "radosgw-admin sync group pipe remove --group-id=group1 --pipe-id=group1-pipe"
+                out = utils.exec_shell_cmd(cmd)
+                cmd = "radosgw-admin sync group flow remove --group-id=group1 --flow-id=group1-flow --flow-type=symmetrical"
+                out = utils.exec_shell_cmd(cmd)
```

Comment:
@udaysk23 Since we are creating and removing sync group pipe, whats the objective here?

Reply:
@TejasC88 Previously we had a bz on RGW crash when we tried to execute sync policy related commands.
We are executing these commands and checking there is crash or not

https://bugzilla.redhat.com/show_bug.cgi?id=2087986

Also above bz is part of close loop automtion

Reply:
@udaysk23 and I had a discussion , and we decided to take this code out to a different py file